modin/pandas/test/utils.py

```diff
@@ -1335,7 +1335,7 @@ def wrapped_function(self, *args, **kwargs):
     return wrapper
 
 
-def generate_dataframe(row_size=NROWS, additional_col_values=None):
+def generate_dataframe(row_size=NROWS, additional_col_values=None, idx_name=None):
```

Comment:
Why do we need this change?

Reply:
To create frames with a named index.

Reply:
I see but why do we need this now? Why is the previous implementation of generate_dataframe not enough?

Reply:
The test test_to_csv() has a new parameter - idx_name. It's used to create a csv with a named index.

Reply:
Got it, thanks!